### PR TITLE
Automated OCP-27666

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei please help to review. Thanks.

### CODE REVIEW

File: lib/rules/web/admin_console/4.4/catalog.xyaml [hunk 1]
```diff
@@ -78,6 +78,10 @@ filter_by_category:
     selector:
       xpath: //li/a[contains(., '<category>')]
     op: click
+filter_by_csv_type:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/catalog.xyaml:81]: This action is not used?

File: lib/rules/web/admin_console/4.5/catalog.xyaml [hunk 2]
```diff
@@ -165,4 +169,13 @@ select_target_namespace:
   action: choose_installmode
   action: click_project_list_dropdown
   action: choose_project_from_dropdown_list
-
+click_item_on_catalog_page:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/catalog.xyaml:172]: I think `click_catalog_item` is much simpler

**Comment by yapei** [comment]
thank you for the update

**Comment by SSshahan** [comment]
@lxia could you help to merge? Thanks

**Comment by liangxia** [comment]
The title is "Automated OCP-27666", but i didn't find any reference of OCP-27666, only rules.

**Comment by SSshahan** [comment]
you mean I need to update the pr title?

**Pull request merged**